x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -62,22 +67,3 @@
     border-right: $euiBorderThin;
   }
 }
-
-/**
- * CSS for hover specific logic
- * It's mildly horrific, so I pulled it out to its own section here
```

Comment:
💯 I'm super glad I pulled this out into its own easily delete-able section. H E L L Y E A